src/main/kotlin/no/nav/syfo/identhendelse/database/IdenthendelseQuery.kt

```diff
@@ -0,0 +1,114 @@
+package no.nav.syfo.identhendelse.database
+
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.application.database.toList
+import no.nav.syfo.domain.Personident
+import java.sql.Connection
+import java.sql.PreparedStatement
+
+const val queryUpdateBehandler =
+    """
+        UPDATE BEHANDLER
+        SET personident = ?
+        WHERE personident = ?
+    """
+
+fun Connection.updateBehandler(
```

Comment:
Personident til Behandler brukes vel i koden som sender dialogmelding... Er det noe vi må tenke på der, rundt hvordan endring av ident fungerer i dialogmelding-land ❓ 🤔

Reply:
Hmm ja, bra catch. Jeg er ikke så stødig på logikken her, hva blir i så fall flyten for feilsituasjonen?

Reply:
Vet egentlig ikke om det vil feile, er ikke så stødig på denne biten her selv... Tenkte kanskje noen som kjenner godt til dialogmeldinger vet om det går bra å sende til behandler som har endret personident (ping @juneih ;))

Reply:
I dag bruker vi ikke PDL for å få tak i fnr til legen, det er EPJ og NHN som gir oss disse. Det er altså ikke nødvendig å oppdatere behandler ved endringer i fnr fra PDL.

Reply:
Okey, da fjerner jeg oppdatering av behandler sin ident 👌🏼